src/js/views/forms/form/form_views.js

```diff
@@ -405,12 +405,30 @@ const UpdateView = View.extend({
 });
 
 const HistoryDroplist = Droplist.extend({
-  viewOptions: {
-    className: 'button-filter',
-    template: hbs`{{far "clock-rotate-left"}}{{formatDateTime updated_at "AT_TIME"}}{{far "angle-down"}}`,
+  viewOptions() {
+    return {
+      className: 'button-filter',
+      template: hbs`
+        {{far "clock-rotate-left"}}{{formatDateTime updated_at "AT_TIME"}} {{formatMessage (intlGet "forms.form.formViews.historyDroplistView.nameText") name=name}}{{far "angle-down"}}
+      `,
+      templateContext() {
+        return {
+          name: this.model.getEditorName(),
+        };
+      },
+    };
   },
-  picklistOptions: {
-    itemTemplate: hbs`{{formatDateTime updated_at "AT_TIME"}}`,
+  picklistOptions() {
+    return {
+      itemTemplate: hbs`
+        {{formatDateTime updated_at "AT_TIME"}}&nbsp;{{formatMessage (intlGet "forms.form.formViews.historyDroplistView.nameText") name=name}}
+      `,
+      itemTemplateContext() {
+        return {
+          name: this.model.getEditorName(),
```

Comment:
I created a `getEditorName()` method on the model to make the code cleaner for this droplist view.

Prior to creating that, both the `viewOptions` and `picklistOptions` here had identical code to get the editor's name:

```js
templateContext() {
  const editor = this.model.get('_editor');
  const name = editor.get('name') || `${ editor.get('first_name') } ${ editor.get('last_name') }`;

  return { name };
}
```

Preferably I wanted to add a `getEditorName()` method to this `HistoryDroplist` view, but it wouldn't let me without borking.